src/pages/gift/Footer.js

```diff
@@ -1,57 +1,29 @@
-import { Container } from 'react-bootstrap';
-
-import PolkadotLogo from '../../images/polkadot_logo_footer.png';
-
-export default function Footer({ selectedAccount }) {
+export default function Footer ({ selectedAccount }) {
   return (
     <footer
-      className="footer"
-      variant="light"
+      className="footer d-flex px-4 py-3"
       expand="md"
-      style={{ backgroundColor: '#F9FAFB' }}>
-      <Container className="d-flex flex-column flex-sm-row">
-        <div style={{ margin: '8px' }}>
-          <a
-            href="https://polkadot.network"
-            target="_blank"
-            rel="noopener noreferrer">
-            <img
-              width={154}
-              className="mr-5"
-              src={PolkadotLogo}
-              alt={'Polkadot'}
-            />
-          </a>
-        </div>
-        <div
-          style={{
-            fontSize: '12px',
-            margin: '8px',
-            maxWidth: 300,
-          }}>
-          <strong>Polkadot Gifts</strong> lets you send DOT gifts to friends and
-          family members.
-        </div>
-        <div className="footer-grow flex-grow-1" />
-        <div
-          style={{
-            fontSize: '12px',
-            margin: '8px',
-            maxWidth: 300,
-          }}>
-          <strong>Never</strong> share your account seed or private key directly
-          with this or any other site.
+    >
+        <div className="d-flex flex-row">
```

Comment:
We need to make it responsive. We can set the direction to column for small screens to stack them vertically for small screens